server/src/controllers/orders/index.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response, NextFunction } from 'express';
+import { Order } from '../../database';
+// import { CustomizedError } from '../../utilities';
```

Comment:
Uncomment this if and only if you follow my suggestions below then you will use it or you will need to remove this line